src/client/stories/button.stories.jsx

```diff
@@ -20,4 +15,4 @@ storiesOf('Button', module)
                 😀 😎 👍 💯
             </span>
         </Button>
-    ));
+    ));
```

Comment:
this was the basic example from storybook yeah? we can just delete this...

Reply:
Done